common/src/main/java/com/gregtechceu/gtceu/common/data/GTMachines.java

```diff
@@ -378,7 +378,7 @@ public static BiConsumer<ItemStack, List<Component>> createTankTooltips(String n
                     .register(),
             ALL_TIERS);
 
-    public final static MachineDefinition[] ENERGY_OUTPUT_HATCH = registerTieredMachines("energy_hatch.output",
+    public final static MachineDefinition[] ENERGY_OUTPUT_HATCH = registerTieredMachines("energy_output_hatch",
```

Comment:
this should be `dynamo_hatch`

Reply:
To keep with consistency, I think it should be maintained to be `energy_input` and `energy_output` respectively.